client/gutenberg/extensions/markdown/jetpack-markdown-block-editor.jsx

```diff
@@ -20,35 +18,32 @@ import MarkdownRenderer from './components/markdown-renderer';
 const PANEL_EDITOR = 'editor';
 const PANEL_PREVIEW = 'preview';
 
+/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Since this seems to be an issue only with one class (`components-toolbar`), perhaps this should be `eslint-disable-next-line` or `eslint-disable-line` instead?

Reply:
Well, this is also the case for the `components-tab-button` class, which in fact makes it 2 instances.

Reply:
Ah, right! Somehow my editor didn't highlight `components-tab-button`. Makes sense to disable linting for the whole function then; I'd still consider adding them as in-liners but not a biggie, just a matter of preference/taste. :-)

Reply:
Thanks for the observations! 

I think a more sustainable solution would be to either update our linter to respect this is an external entry point not part of Calypso, or (not-so-preferred) to disable it for this area of code. But that's separate PR area already 😉